inc/deploygroup_dynamicdata.class.php

```diff
@@ -154,13 +241,18 @@ static function showList($itemtype, $params, $forcedisplay) {
       $_GET['_in_modal'] = true;
```

Comment:
it doesn't seems to be a good idea to set the variable to true because the GLPI header is lost when browsing search results

Reply:
Ok. You are right ! It is obviously a bad idea! I do not even understand how this should run correctly on my test instance 😞